### PR TITLE
Change the project objects to classes

### PR DESCRIPTION
**Pull request by fernandorpm**
# Change the project objects to classes

## To add
- `main_class.js` - the new script file that is being used

## To update
- index.html has changed the script that it calls from `main.js` to `main_class.js`
- the script `main.js` will remain in the files for further comparison

## Requirements
- Use JavaScript classes instead of objects and arrays.
- Create class methods to add and remove books.
- Maintain all of the original functionality of the application.

### CODE REVIEW

**Comment by fernandorpm** [comment]
# To the reviewer
There's an empty CSS file just to guarantee that the repo will pass on the linters

File: main_class.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    const book = new Book(this.idBook, title, author);
+    this.library.push(book);
+
+    localStorage.library = JSON.stringify(this.library);
+
+    this.reloadLibrary();
+
+    inputTitle.value = '';
+    inputAuthor.value = '';
+  }
```
Comment (Davidosky007) [inline main_class.js:42]: * Currently when i try to add a book i get this error in the console kindly fix it 🙏 
Screenshot (149)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @fernandorpm, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
# _Check the comments under the review._ and make the necessary changes 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: main_class.js (hunk 1 above)
Comment (fernandorpm) [reply main_class.js:42]: Hello! Did you type by accident on the main_class.js? It seems that yours contain an `u` somewhere!

**Review by de-scholar — CHANGES_REQUESTED** [review]
Hi @fernandorpm , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] The issue is still there, kindly address it. I used [this link](https://raw.githack.com/fernandorpm/awesome-books/awesome-books-2/index.html) to test your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project, you are left with 2 reviews. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nidalaa** [comment]
Hello @fernandorpm,

_This is an additional review as a result of the second opinion request._

The error experienced by code reviewers is an edge case that we experience in [githack](https://raw.githack.com/).   We double-checked those errors and as edge cases, they should not be a reason to not approve your project.

I can see that you were trying to fix that error in your last commit @fernandorpm  and in the private message screenshot that you provided. Good job! :)

For future reference - I would like to ask you to bring up this kind of issue in PR comments - thanks to that other reviewers who will come to this conversation will have a full context :)

---------

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard @fernandorpm and the next Code Reviewer will mark it as approve right away :)

Happy coding!

**Review by de-scholar — APPROVED** [review]
Hi @fernandorpm , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**